### PR TITLE
Add npm-publish

### PR DESCRIPTION
**Pull request by Pytal**
To be dispatched to @nextcloud namespaced npm package repositories :)

### CODE REVIEW

File: workflow-templates/npm-publish.properties.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "name": "Publish to npm",
+  "description": "Build and publish package to npm.",
```
Comment (skjnldsv) [inline workflow-templates/npm-publish.properties.json:3]:
```suggestion
  "description": "Build and publish package to npm on tagged releases.",
```

**Review by skjnldsv — APPROVED** [review]
Small nitpick, but looks great! :rocket:

File: workflow-templates/npm-publish.yml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+
+  publish:
+    if: env.NODE_AUTH_TOKEN != null && github.repository_owner == 'nextcloud-releases'
+    runs-on: ubuntu-latest
+
+    name: Build and publish to npm
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
```
Comment (skjnldsv) [inline workflow-templates/npm-publish.yml:23]:
```suggestion
    runs-on: ubuntu-latest

    name: Build and publish to npm
    steps:
      - name: Check actor permission
        uses: skjnldsv/check-actor-permission@v2
        with:
          require: admin

      - name: Checkout
        uses: actions/checkout@v2
```

**Review by skjnldsv — CHANGES_REQUESTED** [review]

File: workflow-templates/npm-publish.properties.json (hunk 1 above)
Comment (Pytal) [reply workflow-templates/npm-publish.properties.json]: Added :)

File: workflow-templates/npm-publish.yml (hunk 2 above)
Comment (Pytal) [reply workflow-templates/npm-publish.yml]: Thanks, changes made :)

**Review by skjnldsv — APPROVED** [review]

**Pull request merged**